packages/atlas/src/MainLayout.tsx

```diff
@@ -53,6 +54,20 @@ export const MainLayout: FC = () => {
     onExitClick: () => closeDialog(),
   })
 
+  useEffect(() => {
```

Comment:
We should return when the sentry is not set in atlas.config.yml

Reply:
good catch, added this.